.vib/zookeeper/goss/goss.yaml

```diff
@@ -0,0 +1,12 @@
+gossfile:
+  # Goss tests exclusive to the current container
+  ../../zookeeper/goss/zookeeper.yaml: {}
+  # Load scripts from .vib/common/goss/templates
+  ../../common/goss/templates/check-binaries.yaml: {}
```

Comment:
You forgot to include the `check-app-version` test:

```
$ zkServer.sh version
...
Apache ZooKeeper, version 3.8.1 2023-01-25 16:31 UTC
```

Reply:
Added, thanks!